index.html

```diff
@@ -22,16 +22,13 @@ <h1 class="primary-text">
                 Hey There. <br>I'm Stephen.
             </h1>
             <h3 class="supporting-text">I'm a Software Developer</h3>
-            <p class="description">I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don't hesitate to contact me.</p>
+            <p class="description">I can help you build a product , feature or website. Look through some of my work and experience! <br> If you like what you see and have a project you need coded, don't hesitate to contact me.</p>
```

Comment:
[OPTIONAL]
- I see your fluffy and colorful creativity hat here 😄. But since you are already exposed to the doctrines of CSS how that it can create space anywhere, change the appearance of anything, to name a few of its strengths, I think it'll show your css mastery more gracefully using css to achieve this or better still, creating a new paragraph - which seems to be a more correct approach, depending on whether the statement is a continuum of the latter.
Note: This is not a compulsory request, just a suggestion.